tests/improver-apply-emos-coefficients/02-gaussian.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+. $IMPROVER_DIR/tests/lib/utils
+
+@test "apply-emos-coefficients for diagnostic with assumed gaussian distribution" {
+  improver_check_skip_acceptance
+  KGO="apply-emos-coefficients/gaussian/kgo.nc"
```

Comment:
Why have a separate KGO to the version in the ensemble-calibration/gaussian/kgo.nc? Are the tests not set up to give the same result? (nccmp-ing the two they appear to be very nearly identical, precision level differences). It would be nice to use one KGO for both ensemble-calibration and this CLI as it would demonstrate equivalence between the one step and two step processes of using ensemble calibration.

Reply:
I've updated these CLI tests now so that now there isn't any specific `apply-emos-coefficients` CLI test output, as instead this CLI uses inputs and known good outputs that already exist for the `estimate-emos-coefficients` and `ensemble-calibration` CLIs. This shows the equivalency of either running the `ensemble-calibration` CLI that estimates the coefficients and applies them within one CLI versus the combination of the `estimate-emos-coefficients` and `apply-emos-coefficients`.

Reply:
Thanks.